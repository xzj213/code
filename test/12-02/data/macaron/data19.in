14
528 0 528 528 528 455 455 528 0 455 455 0 0 455
0 528 0 455 528 455 528 455 528 528 455 455 528 455
0 0 528 455 455 528 0 0 0 455 455 455 455 455
528 455 0 455 455 528 136 455 528 0 528 528 0 455
0 528 455 0 528 455 0 455 455 455 0 455 0 528
528 528 528 455 0 455 528 0 528 528 455 0 528 455
455 528 528 528 528 455 0 528 528 528 455 0 528 0
528 455 0 0 455 0 0 528 455 528 455 0 0 528
455 455 218 455 455 528 455 528 455 455 0 0 455 0
0 0 0 455 0 455 528 0 528 0 528 528 528 455
528 455 0 0 455 0 528 455 528 0 528 528 455 455
455 455 0 0 455 528 0 508 0 0 0 528 0 0
0 0 528 0 528 528 455 0 0 0 528 528 455 528
455 455 528 528 0 455 455 528 0 0 455 0 0 528
