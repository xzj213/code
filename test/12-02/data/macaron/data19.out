0 59 82 105 136 179 218 225 285 294 335 372 405 430 455 508 528 555 578 633 664 675 714 753 781 822 863 868 901 958 983 1004 
